packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md

```diff
@@ -0,0 +1,110 @@
+### 1: Start password recovery
+
+The user starts the password recovery flow with the following steps:
+
+1. Clicks the **Forgot Password?** link on the sign-in page.
+2. Enters their **Email or Username** in the dialog and then clicks **Next**.
+3. Chooses **Email** as the authenticator they want to use for password recovery and clicks **Submit**.
```

Comment:
Same comment as above, this isn't relevant unless it's something the developer does. And if so, refere to end-user doc.